crt_portal/cts_forms/views.py

```diff
@@ -336,6 +337,38 @@ def serialize_data(report, request, report_id):
     return output
 
 
+class ProfileView(LoginRequiredMixin, FormView):
+    """Can be used for updating section filter for a profile"""
+    form_class = ProfileForm
+
+    def has_related_profile(self, user):
+        return hasattr(user, 'profile') and user.profile is not None
+
+    def post(self, request):
+        """Update or create Profile"""
+        if hasattr(request.user, 'profile') and request.user.profile is not None:
+            instance = request.user.profile
+        else:
+            instance = Profile()
+            instance.user = request.user
```

Comment:
This works but all of the evaluation is handled by `hasattr`, Opportunity here to shorten this with something like
```suggestion
        if hasattr(request.user, 'profile'):
            instance = request.user.profile
        else:
            instance = Profile(user=request.user)
```